packages/eas-cli/src/utils/code-signing.ts

```diff
@@ -103,66 +104,26 @@ export async function getKeyAndCertificateFromPathsAsync({
   };
 }
 
-export type MultipartPart = { headers: Map<string, string>; body: string };
+function convertArrayBufferToBuffer(ab: ArrayBuffer): Buffer {
+  const buf = Buffer.alloc(ab.byteLength);
+  const view = new Uint8Array(ab);
+  for (let i = 0; i < buf.length; ++i) {
+    buf[i] = view[i];
+  }
+  return buf;
```

Comment:
I would expect something like this to work:
```suggestion
  return Buffer.from(new Uint8Array(ab));
```
https://nodejs.org/api/buffer.html#static-method-bufferfrombuffer

or just inline the code